### PR TITLE
Common/header changes to files in dir/styling/rwd

### PR DESCRIPTION
**Pull request by adrianSACHA**
fixed active navigation and RWD.
Please review :)

### CODE REVIEW

File: src/common/Navigation/Search/styled.js [hunk 1]
```diff
@@ -23,22 +25,32 @@ export const MainSearch = styled.div`
   background: ${({ theme }) => theme.colors.white};
   border-radius: 33px;
   border: 1px solid ${({ theme }) => theme.colors.mystic};
+  height: 48px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}px) {
+    height: 44px;
+    width: 80%;
+    /* gap: 8px; */
```
Comment (konradkapica) [inline src/common/Navigation/Search/styled.js]: is this commented line necessary?

File: src/common/Navigation/styled.js [hunk 2]
```diff
@@ -1,22 +1,42 @@
...
   max-width: 100%;
   background: ${({ theme }) => theme.colors.black};
   color: ${({ theme }) => theme.colors.white};
   display: flex;
   flex-wrap: wrap;
-  justify-content: center;
   align-items: center;
+  justify-content: space-between;
+  height: 94px;
+
```
Comment (konradkapica) [inline src/common/Navigation/styled.js:29]: empty line

File: src/common/Navigation/styled.js [hunk 3]
```diff
@@ -1,22 +1,42 @@
...
+  gap: 5px;
 
   @media (max-width: ${({ theme }) => theme.breakpoints.mobileMin}px) {
-    margin: 0;
-    gap: 5px;
+    /* margin: 0; */
+    /* gap: 5px; */
```
Comment (konradkapica) [inline src/common/Navigation/styled.js]: should you keep it or remove?

**Review by konradkapica — CHANGES_REQUESTED** [review]
Good job Adrian! 
Theres a few comments here, I think if those lines are not necessary you should delete them.

File: src/common/Navigation/Search/styled.js (hunk 1 above)
Comment (adrianSACHA) [reply src/common/Navigation/Search/styled.js]: fixed :)

File: src/common/Navigation/styled.js (hunk 2 above)
Comment (adrianSACHA) [reply src/common/Navigation/styled.js:29]: fixed

File: src/common/Navigation/styled.js (hunk 3 above)
Comment (adrianSACHA) [reply src/common/Navigation/styled.js]: changed it

**Comment by adrianSACHA** [comment]
Changed styles,
added breakpoint for responsiveness.

Please review it ;)

**Review by konradkapica — APPROVED** [review]
Great!

**Pull request merged**